sdk/include/opentelemetry/sdk/logs/logger.h

```diff
@@ -67,9 +67,10 @@ class Logger final : public opentelemetry::logs::Logger
   // The name of this logger
   std::string logger_name_;
 
-  // The logger context of this Logger. Uses a weak_ptr to avoid cyclic dependency issues the with
-  std::weak_ptr<LoggerContext> context_;
   std::unique_ptr<instrumentationlibrary::InstrumentationLibrary> instrumentation_library_;
```

Comment:
The order of initialization in the ctor has to be changed too.

Reply:
Fixed.